core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -107,6 +96,7 @@
     final String transportPrimaryAction;
     final String executor;
     final boolean checkWriteConsistency;
```

Comment:
I moved all of them to private with the exception of the services, which I think are handy and should/can be used by subclasses. I also removed some unused varaibles